lib/widgets/forms/task_form.dart

```diff
@@ -0,0 +1,556 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_picker/flutter_picker.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:smart_select/smart_select.dart';
+import 'package:mongo_dart/mongo_dart.dart' as Mongo;
```

Comment:
Btw any particular reason you are doing the named imports for mongo?

Reply:
Can't remember which, but it has come class that overlaps with Material class. It will probably go anyway. I think is only for mock data.